README.md

```diff
@@ -54,3 +55,6 @@ License
 -------
 
 Akka is Open Source and available under the Apache 2 License.
+
+[akka-dependency-badge]: https://index.scala-lang.org/count.svg?q=dependencies%3Aakka%2F*&subject=scaladex:&color=blue&style=flat-square "Built with Akka"
```

Comment:
you can leave the link decoded like: 
https://index.scala-lang.org/count.svg?q=dependencies:akka/*&subject=scaladex:&color=blue&style=flat-square